releases/release-1.24/exceptions.yaml

```
@@ -0,0 +1,22 @@
+# Exception requests in 1.24
+# Google Group: https://groups.google.com/forum/#!forum/kubernetes-sig-release
+# Release Team Lead: James Laverack ([@JamesLaverack](https://github.com/JamesLaverack) 
+
+# Enhancements Freeze Exceptions requested in 1.24
+- name: "Add support for resizing PVs"
+  issue: 284
+  date_requested: 2022-02-03
+  date_reviewed: 2022-02-03
+  thread: "https://groups.google.com/a/kubernetes.io/g/release-team/c/x6AlKFEQU4A"
+  pull_requests:
+    - "https://github.com/kubernetes/enhancements/pull/284"
```

```suggestion
    - "https://github.com/kubernetes/enhancements/pull/3210"
```
This should be the PR that introduced the fix.